backend/src/main/java/de/pddao/backend/model/PollenItem.java

```diff
@@ -0,0 +1,18 @@
+package de.pddao.backend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class PollenItem {
+    private String englishName;
+    private String latinName;
+    private String beginSeason;
```

Comment:
do you always have a month here? Maybe than use some thing like [Month](https://docs.oracle.com/javase/8/docs/api/java/time/Month.html)